classes/class-wc-rest-connect-account-settings-controller.php

```diff
@@ -34,6 +34,7 @@ public function get() {
 			'formData' => $this->settings_store->get_account_settings(),
 			'formMeta' => array(
 				'can_manage_payments' => $this->can_user_manage_payment_methods(),
+				'can_edit_settings' => true,
```

Comment:
We need the `can_edit_settings` to maintain backwards-compatibility in Calypso. So:
* `can_manage_payments: true`: Show the whole UI, the user can do whatever.
* `can_manage_payments: false`: Disable the whole UI, the user can't edit payments or anything else, because this is an old version of the plugin that will fail if he tries to save *anything*.
* `can_manage_payments: false, can_edit_settings: true`: Disable the payment UI but keep the rest of the UI, since this is a new version of the plugin that will admit changes in the non-payment label settings.